.github/pull_request_template.md

```diff
@@ -0,0 +1,74 @@
+<!--
+Thank you for your pull request. Please review the requirements below.
+
+Bug fixes and new features should be reported on the issue tracker: https://github.com/lif-initiative/lif-core/issues
+
+Contributing guide: https://github.com/lif-initiative/lif-core/blob/main/docs/contributing.md
+Code of Conduct: https://github.com/lif-initiative/lif-core/blob/main/CODE_OF_CONDUCT.md
+-->
+
+##### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] commit message follows commit guidelines (see commitlint.config.mjs)
+- [ ] tests are included (unit and/or integration tests)
+- [ ] documentation is changed or added (in /docs directory)
+- [ ] code passes linting checks (`uv run ruff check`)
+- [ ] code passes formatting checks (`uv run ruff format`)
+- [ ] code passes type checking (`uv run ty check`)
+- [ ] pre-commit hooks have been run successfully
+- [ ] database schema changes: migration files created and CHANGELOG.md updated
+- [ ] API changes: base (Python code) documentation and project README updated
+- [ ] configuration changes: relevant folder README updated
+- [ ] breaking changes: added to MIGRATION.md with upgrade instructions
+      and CHANGELOG.md entry
```

Comment:
```suggestion
- [ ] breaking changes: added to MIGRATION.md with upgrade instructions and CHANGELOG.md entry
```

Reply:
I run a markdownlinter that warns on lines longer that 80 characters.